controllers/download_controller.coffee

```diff
@@ -33,4 +34,23 @@ router.get '/download', (req, res) ->
 
 	return request.get(options).pipe(res)
 
+router.get '/download/:image', (req, res) ->
+	image = req.params.image
+
+	if image is 'Z7010'
+		imageName = 'resin-supercomputer-0.1.0-0.0.14-Z7010-16.img'
+	else if image is 'Z7020'
+		imageName = 'resin-supercomputer-0.1.0-0.0.14-Z7020-16.img'
+	else
+		return res.sendStatus(404)
+
+	urlPrefix = config.s3.url + 'os/'
```

Comment:
The client was adding the `os/` suffix to the URL: https://github.com/supercomputer-io/parallellocalypse-hub/blob/master/public/js/controllers/download_controller.coffee#L18